plugins/CoreHome/Columns/VisitLastActionDayOfWeek.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * Piwik - free/libre analytics platform
+ *
+ * @link http://piwik.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ *
+ */
+namespace Piwik\Plugins\CoreHome\Columns;
+
+use Piwik\Columns\DimensionMetricFactory;
+use Piwik\Columns\MetricsList;
+use Piwik\Plugin\Dimension\VisitDimension;
+use Piwik\Metrics\Formatter;
+use function Piwik\Plugins\VisitTime\translateDayOfWeek;
```

Comment:
`use function` is available in PHP 5.6+ only, but we still support 5.5.9+. 
See https://secure.php.net/manual/en/language.namespaces.importing.php

Reply:
Thanks, fixed! Didn't notice it. Will have an eye on the tests once they are finished.